contract_compassion/model/contracts.py

```diff
@@ -60,6 +60,11 @@ class recurring_contract(models.Model):
         'Partner Contract Number', required=True)
     end_reason = fields.Selection('get_ending_reasons')
     end_date = fields.Date(readonly=True, track_visibility='onchange')
+    paid_months = fields.Integer(
+        compute='_set_months_paid',
+        readonly=True,
+        string='Mois payés',

```

Comment:
field strings should not be in French. You can write in English and translation is done with .po language files (you don't have to take care of it).